tests/SprykerEcoTest/Zed/Adyen/Business/FacadeTest.php

```diff
@@ -301,4 +301,24 @@ public function testHandleCreditCard3dResponseFromAdyen(): void
             $this->assertNotEmpty($paymentAdyenOrderItem->getSpyPaymentAdyen()->getPspReference());
         }
     }
+
+    /**
+     * @return void
+     */
+    public function testHandleOnlineTransferResponseFromAdyenWithRefusedStatus(): void
+    {
+        //Arrange
+        $facade = $this->createFacade($this->createAdyenApiFacade());
+        $orderTransfer = $this->setUpCommandTest(
+            static::PROCESS_NAME_ADYEN_CREDIT_CARD,
+            static::REDIRECT_RESPONSE_RESULT_CODE_REFUSED,
+        );
+        $redirectResponseTransfer = $this->createRefusedRedirectResponseTransfer($orderTransfer);
+
+        //Act
+        $result = $facade->handleOnlineTransferResponseFromAdyen($redirectResponseTransfer);
```

Comment:
result naming is unacceptable
facade returns `AdyenRedirectResponseTransfer`
what is the reason you use `result`?